src/servers/src/http/authorize.rs

```diff
@@ -156,9 +145,12 @@ pub fn extract_catalog_and_schema<B>(request: &Request<B>) -> (&str, &str) {
         .and_then(|header| header.to_str().ok())
```

Comment:
Let's leave header logic as is. If user specifies this header, it's most likely intentionally.